pages/nft-book-store/index.vue

```diff
@@ -289,31 +294,57 @@ watch(tableRowsPerPage, () => {
   tablePage.value = 1
 })
 
-onMounted(async () => {
-  let { data, error: fetchError } = await useFetch(`${LIKE_CO_API}/likernft/book/store/list?wallet=${wallet.value}`,
+async function fetchBookList (params: { key?: number, limit?: number } = {}) {
+  const qsPayload: any = {
+    wallet: wallet.value,
+    limit: params.limit || 100
+  }
+  if (params.key) {
+    qsPayload.key = params.key
+  }
+  const { data, error: fetchError } = await useFetch(`${LIKE_CO_API}/likernft/book/store/list?${Object.entries(qsPayload).map(([key, value]) => `${key}=${value}`).join('&')}`,
```

Comment:
How about using `URLSearchParams.toString()` to format qs?

Reply:
Merged and update later